charts/nri-prometheus/templates/configmap.yaml

```
@@ -8,6 +8,10 @@ apiVersion: v1
 data:
   config.yaml: |
     cluster_name: {{ include "nri-prometheus.cluster" . }}
-{{- if .Values.config }}
-{{ toYaml .Values.config | indent 4 -}}
-{{ end }}
+{{- if .Values.guidedInstall }}
+    {{- .Files.Get "static/guidedinstalldefaults.yaml" | indent 4 -}}
+{{ else }}
+    {{- if .Values.config }}
+    {{ toYaml .Values.config | indent 4 | trim  -}}
+    {{ end }}
+{{ end }}
```

It should be possible to compact else and if
```suggestion
{{- else if .Values.config }}
    {{ toYaml .Values.config | indent 4 | trim }}
{{- end }}

```